pypeit/core/coadd1d.py

```diff
@@ -72,48 +68,44 @@ def new_wave_grid(waves, wave_method='iref',iref=0, wave_grid_min=None, wave_gri
         wave_grid (ndarray): New wavelength grid, not masked
     """
 
-    wave_mask = waves>1.0
+    c_kms = constants.c.to('km/s').value
+
+    if masks is None:
+        masks = waves > 1.0
 
     if wave_grid_min is None:
-        wave_grid_min = np.min(waves[wave_mask])
+        wave_grid_min = waves[masks].min()
```

Comment:
My understanding is that these are aliased to the same function, and so it does not matter. I somewhat prefer using the attribute min function then np.min since it is more compact.